pkg/cmd/server/api/v1/types.go

```diff
@@ -58,6 +60,9 @@ type NodeConfig struct {
 	// command line arguments.  These are not migrated or validated, so if you use them they may become invalid.
 	// These values override other settings in NodeConfig which may cause invalid configurations.
 	KubeletArguments ExtendedArguments `json:"kubeletArguments,omitempty"`
+
+	// SyncPeriod is how often iptable rules are refreshed
+	SyncPeriod time.Duration `json:"syncPeriod"`
```

Comment:
a lot of things sync... if this only controls iptables, it should be named more specifically